src/Tokenizer/Analyzer/ClassyAnalyzer.php

```diff
@@ -57,6 +57,10 @@ public function isClassyInvocation(Tokens $tokens, int $index): bool
             return true;
         }
 
+        if (\PHP_VERSION_ID >= 8_00_00 && $nextToken->equals(')') && $prevToken->equals('(') && $tokens[$tokens->getPrevMeaningfulToken($prev)]->isGivenKind(T_CATCH)) {
```

Comment:
It won't work for `(/* non-capturing catch */ Exception /* just because! */)`, you need to use `getPrevMeaningfulToken` / `getNextMeaningfulToken` for that. Please add such test cases too.

Reply:
It looks like it works because $nextToken and $prevToken are already meaningful tokens. I added tests.

Reply:
Ah, my bad, I don't know why I read this like it was forwarding/rewinding to next/prev token 😅. Test cases with comments were needed anyway, so 🤷‍♂️.

Reply:
@fancyweb I see you added test cases with comments also for other scenarios, but these comments are not correct sometimes - it's _not_ non-capturing catch if there's variable 😉. Can we change it to something like `(/* ... */ Exception $e /* ... */)`?

Reply:
Yes, it's done!